src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelist.java

```diff
@@ -79,6 +81,30 @@ public StaticWhitelist(String... lines) throws IOException {
         this(asList(lines));
     }
 
+    /**
+     * Filters a line, returning the content that must be processed.
+     * @param line Line to filter.
+     * @return {@code null} if the like must be skipped or the content to process if not.
+     */
+    static String filter(String line) {
+        if (line == null) {
+            return null;
```

Comment:
:ant: AFAICT both callers always pass non-null, so delete this and make parameter `@Nonnull`